scout/server/blueprints/cases/controllers.py

```diff
@@ -329,12 +329,19 @@ def case(store, institute_obj, case_obj):
             for img in case_obj["custom_images"][img_section]:
                 img["data"] = b64encode(img["data"]).decode("utf-8")
 
+    # Limit secondary findings according to institute settings
+    limit_genes = store.safe_genes_filter(institute_obj["_id"])
+
     data = {
         "institute": institute_obj,
         "case": case_obj,
         "status_class": STATUS_MAP.get(case_obj["status"]),
-        "other_causatives": [var for var in store.check_causatives(case_obj=case_obj)],
-        "managed_variants": [var for var in store.check_managed(case_obj=case_obj)],
+        "other_causatives": [
+            var for var in store.check_causatives(case_obj=case_obj, limit_genes=limit_genes)
```

Comment:
nicely prepared! 😁

Reply:
I'm just wondering if the search and subsequent loading of case page will be slower, especially when having a long list of limit_genes. Have you tried it on stage? I haven't

Reply:
I was thinking the same, but then assumed you had tested and checked for index use! 😅

Reply:
I did try it with multiple panels, and it didn't grind to a halt or anything.

Reply:
But do check that it hits indexes!

Reply:
`check_causatives` is first looking into the events to collect causatives and partial causatives, then loops thru them to see if they meet the requirements.

`check_managed` is using the positional_variant_ids to collect all variants that match. We should be fine I think

Reply:
Ok, so could slow down with increasing counts of relevant events then, but still, it is matching all variants first, so as long as the gene check is done only after match to the variant, it must be a very small contribution. Things should be all green.